static/js/bot_data.js

```diff
@@ -13,7 +13,7 @@ const send_change_event = _.debounce(function () {
 const set_can_admin = function bot_data__set_can_admin(bot) {
     if (page_params.is_admin) {
         bot.can_admin = true;
-    } else if (bot.owner !== undefined && people.is_current_user(bot.owner)) {
+    } else if (bot.owner && people.is_current_user(bot.owner)) {
```

Comment:
Hmm, actually, reading the code now, I think this code should ideally looking at `bot.bot_owner_id` and using `people.is_current_user_id`, not something based on emails?  This is coming from the "bots" data set, so I'm not sure whether this is code we neglected to migrate on the backend or frontend.

Reply:
(We've had several bugs of this form since migrating to `bot_owner_id` in the main `people` data structure, so this might be another).  @Hypro999 FYI in case you remember looking at this when you did that migration.